README.md

```diff
@@ -1,76 +1,39 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio: setup and mobile version skeleton
 
```

Comment:
- [ ] Change your title to Personal portfolio instead of setup and mobile version skeleton because after this task you will have to implement other tasks in the same project.